plugins/modules/sonic_radius_server.py

```diff
@@ -132,7 +134,7 @@
       - In case of merged, the input mode configuration will be merged with the existing radius server configuration on the device.
       - In case of deleted the existing radius server mode configuration will be removed from the device.
```

Comment:
Please add descriptions for replaced and overridden

Reply:
Added. Thanks!